### PR TITLE
Make staff names on the detailed match page clickable like the regular match page

### PR DESCRIPTION
**Pull request by ZusorCode**
Replaces this
image
with this
image

### CODE REVIEW

**Comment by slmnio** [comment]
Beautiful - this was on my mental list to do, and you've even used the cute little LinkedPlayers component just as I would have

File: website/src/components/website/DetailedMatchStat.vue [hunk 1]
```diff
@@ -3,21 +3,25 @@
...
+            <LinkedPlayers :players="match[data] || override" />
+        </div>
         <div class="stat-b" v-else-if="override">{{ format ? format(override) : override }}</div>
         <div class="stat-b" v-else>{{ format ? format(match[data]) : match[data] }}</div>
 
     </div>
 </template>
 
 <script>
+import { url } from "@/utils/content-utils";
```
Comment (slmnio) [inline website/src/components/website/DetailedMatchStat.vue:16]: Don't think you need the url function in here anymore

Comment (ZusorCode) [reply website/src/components/website/DetailedMatchStat.vue]: Oops, I imported that but then didn't end up using it. Fixed!

**Review by slmnio — APPROVED** [review]

**Comment by ZusorCode** [comment]
Oh, and can you add the `hacktoberfest-accepted` label to this pr?

**Comment by slmnio** [comment]
enjoy your points or whatever it is :)

**Pull request merged**